### DISCUSSION TITLE
Are FileField and ImageField supported?

### DISCUSSION THREAD

**Discussion by marcfargas** [viewflow/viewflow#411] [General]
Hi there,

It is unclear from the demo, but I tried creating a model with either a `FileField` or `ImageField` and in both cases the field will not show up in the form, with "Unprocessable Entity: /...." logged on Submit.

The example [here](https://blog.devgenius.io/bridging-human-intuition-with-ai-crafting-precise-video-summaries-with-viewflow-chatgpt-and-2e997eebbfc7) made me think it would work, but maybe I'm mistaken.

The test was with default `ModelViewSet`.

**Comment by kmmbvnr** [comment]
Yep, indeed, it wan't included into Viewflow open source version, fixed

Thank you for the report
